packages/orchestrator/lib/routes/v1/tasks/taskId/getOutput.ts

```diff
@@ -55,7 +51,7 @@ const handler = (scheduler: Scheduler, eventEmitter: EventEmitter) => {
             cleanupAndRespond((res) => res.status(200).json({ state: completedTask.state, output: completedTask.output }));
         };
         const timeout = setTimeout(() => {
-            cleanupAndRespond((res) => res.status(204).send());
+            cleanupAndRespond((res) => res.status(500).send({ error: { code: 'timeout', message: 'Long polling timeout' } }));
```

Comment:
There is an issue with the API types and express. Since the response body can actually be empty, we can get an undefined result even though the compiler said that's not the case. That's the source of the `mapError` called on undefined. I will fix in another PR. 
Anyway sending 204 is not a very good solution and it is better to explicitely differentiate an empty response vs a timeout